pkg/reconciler/events/storage/controller.go

```diff
@@ -60,19 +61,24 @@ func NewController(
 		storageLister:  cloudstoragesourceInformer.Lister(),
 		createClientFn: gstorage.NewClient,
 	}
-	impl := controller.NewImpl(r, r.Logger, reconcilerName)
+	impl := cloudstoragesourcereconciler.NewImpl(ctx, r)
 
 	r.Logger.Info("Setting up event handlers")
 	cloudstoragesourceInformer.Informer().AddEventHandlerWithResyncPeriod(controller.HandleAll(impl.Enqueue), reconciler.DefaultResyncPeriod)
 
+	// Call GlobalResync on pubsubsource.
+	grCh := func(obj interface{}) {
+		impl.GlobalResync(cloudstoragesourceInformer.Informer())
+	}
+
```

Comment:
same with the other PR, no need for this

Reply:
done